stable/kibana/templates/configmap.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
 {{- range $key, $value := .Values.files }}
-  {{ $key }}: |
-{{ toYaml $value | default "{}" | indent 4 }}
+  {{- if $value }}
+    {{ $key }}: |
+    {{ tpl (toYaml $value | default "{}" | indent 4) $ }}
+  {{- end}}
```

Comment:
Can we outdent here so that there is a coherence across the chart?

Reply:
Not sure which one it should be:

```yaml
{{- range $key, $value := .Values.files }}
{{- if $value }}
{{ $key }}: |
{{ tpl (toYaml $value | default "{}" | indent 4) $ }}
{{- end}}
{{- end -}}
```
or

```yaml
{{- range $key, $value := .Values.files }}
{{- if $value }}
  {{ $key }}: |
  {{ tpl (toYaml $value | default "{}" | indent 4) $ }}
{{- end}}
{{- end -}}
```

Reply:
The second one. Just with the range and if outdented to the edge of the line.